Drink, Ingredient, and Order Models created

recneps121 opened the pull request.

logan-nielsen commented:
Looks good! I just have two minor comments before we merge this.
1. When I run the Django server I'm getting an error that says "No module named djmoney". Are you using the [django money](https://pypi.org/project/django-money/) package in your changes? If so I would add that to the requirements.txt file in the app/ directory with the version number you are using. The version can be easily found using the pip show command
2. Assuming I'm understanding correctly I don't think we need the canMake field in the Drink class. I think we can just compute this dynamically when it is needed based on the inventory and the store's current money rather than updating a database field every time an order is made. If you have a reason for doing this just let us know, these are just my thoughts.

logan-nielsen reviewed (changes requested):
Looks good! I just have two minor comments before we merge this.
1. When I run the Django server I'm getting an error that says "No module named djmoney". Are you using the [django money](https://pypi.org/project/django-money/) package in your changes? If so I would add that to the requirements.txt file in the app/ directory with the version number you are using. The version can be easily found using the pip show command
2. Assuming I'm understanding correctly I don't think we need the canMake field in the Drink class. I think we can just compute this dynamically when it is needed based on the inventory and the store's current money rather than updating a database field every time an order is made. If you have a reason for doing this just let us know, these are just my thoughts.

recneps121 commented:
Alright I removed the can_make field from Drink and added django-money to the requirements file. Let me know if there's anything else you can think of.
